template/clients/base/scripts/generate-clients.mjs.njk

```diff
@@ -11,13 +11,15 @@ import { renderVisitor as renderRustVisitor } from '@kinobi-so/renderers-rust';
 import { getAllProgramIdls } from './utils.mjs';
 
 // Instanciate Kinobi.
-const [idl, ...additionalIdls] = getAllProgramIdls().map(idl => rootNodeFromAnchor(require(idl)))
+const [idl, ...additionalIdls] = getAllProgramIdls().map((idl) =>
+  rootNodeFromAnchor(require(idl))
+);
 const kinobi = k.createFromRoot(idl, additionalIdls);
 
 // Update programs.
 kinobi.update(
   k.updateProgramsVisitor({
-    '{{ programCrateName | camelCase }}': { name: '{{ programName | camelCase }}' },
+    {{ programCrateName | camelCase }}: { name: '{{ programName | camelCase }}' },
```

Comment:
Did you mean to shed these single quotes here?

Reply:
Yeah I took that opportunity to ensure the JS formatting was correct on the scripts as well. Since this one is a `.njk` template, Prettier doesn't trigger on the generator so it's a manual process. But as a result, after generating the repo and going inside that file, saving it will no longer trigger git changes.

This particular change it to replace `{ 'myProgram': {} }` with `{ myProgram: {} }`.